llama_stack/cli/stack/run.py

```diff
@@ -130,20 +131,45 @@ def _run_stack_run_cmd(self, args: argparse.Namespace) -> None:
         except AttributeError as e:
             self.parser.error(f"failed to parse config file '{config_file}':\n {e}")
 
-        run_args = formulate_run_args(args.image_type, args.image_name, config, template_name)
-
-        run_args.extend([str(config_file), str(args.port)])
-        if args.disable_ipv6:
-            run_args.append("--disable-ipv6")
-
-        for env_var in args.env:
-            if "=" not in env_var:
-                self.parser.error(f"Environment variable '{env_var}' must be in KEY=VALUE format")
-            key, value = env_var.split("=", 1)  # split on first = only
-            if not key:
-                self.parser.error(f"Environment variable '{env_var}' has empty key")
-            run_args.extend(["--env", f"{key}={value}"])
-
-        if args.tls_keyfile and args.tls_certfile:
-            run_args.extend(["--tls-keyfile", args.tls_keyfile, "--tls-certfile", args.tls_certfile])
-        run_with_pty(run_args)
+        # If the image type and name are not provided, we assume the user wants to run using system
+        # packages and directly run the server code.
+        if not args.image_type and not args.image_name:
+            logger.info("No image type or image name provided. Assuming system packages.")
+            # Import and run the server directly
+            try:
+                from llama_stack.distribution.server.server import main as server_main
+
+                # Remove the first argument which is the command name, the second argument which is
+                # the config file, the third argument which is the pass to the config file.
```

Comment:
isn't the "second argument" `stack` (and the third `run`) here? (and the forth - that you also skip - the config file)?

Reply:
Changed the entire approach PTAL!